v2/spiffeid/trustdomain.go

```diff
@@ -99,3 +80,19 @@ func (td TrustDomain) IsZero() bool {
 func (td TrustDomain) Compare(other TrustDomain) int {
 	return strings.Compare(td.name, other.name)
 }
+
+// isValidTrustDomainChar returns true if the given character is acceptable in
+// a path segment. Acceptable trust domain characters consist of the same set
```

Comment:
"... is acceptable in a trust domain name."

Reply:
:+1: